nextcord/enums.py

```diff
@@ -1947,6 +1952,11 @@ class AutoModerationTriggerType(IntEnum):
 
     keyword = 1
     """This rule checks if content contains words from a user defined list of keywords."""
+    member_update = 2
```

Comment:
This is not a trigger type. Member profiles is covered in type `6`.
![image](https://github.com/user-attachments/assets/421e0404-3899-424b-8640-d0ba8039488f)